precompiles/staking/events_test.go

```diff
@@ -1,6 +1,7 @@
 package staking_test
 
 import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
there's a format issue

Reply:
I forgot to execute `make format` before submitting. Already done.

In addition, other precompiled contracts have similar problems. I would like to ask if there is no review of the previous code?